nav2_simple_commander/nav2_simple_commander/robot_navigator.py

```diff
@@ -191,7 +206,8 @@ def spin(self, spin_dist=1.57, time_allowance=10):
         goal_msg.time_allowance = Duration(sec=time_allowance)
 
         self.info(f'Spinning to angle {goal_msg.target_yaw}....')
-        send_goal_future = self.spin_client.send_goal_async(goal_msg, self._feedbackCallback)
+        send_goal_future = self.spin_client.send_goal_async(
```

Comment:
why change? go through this PR. All changes that did not need to be made, revert